<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="channelComBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>channel0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>channel1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>channel2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>channel3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>channel4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="channelLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Channels:</string>
    </property>
   </widget>
   <widget class="QLabel" name="commandLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Command:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="commandComBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>30</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>start_measure</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>set_range</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>stop_measure</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>get_status</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>get_result</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="outputBrowser">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>120</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="rangeLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ranges</string>
    </property>
   </widget>
   <widget class="QComboBox" name="rangeComBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>range0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>range1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>range2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>range3</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
